Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -58,7 +58,7 @@
   $SEC101/022.PayPalBraintreeAccessToken=access_token\$production\$(?P<secret>[0-9a-z]{16}\$[0-9a-f]{32})
   $SEC101/023.AmazonMwsAuthToken=(?:[^a]|^)(?P<secret>amzn\\.mws\\.[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})(?:[^0-9a-f]|$)
   $SEC101/024.TwilioApiKey=(?:[^S]|^)(?P<secret>SK(?i)[0-9a-f]{32})(?:[^0-9a-f]|$)
-  $SEC101/028.PlaintextPassword=(?i)ConvertTo-SecureString\s*(?:-[S]tring)?\s*["'](?P<secret>[^"')(?:]*)["']
+  $SEC101/028.PlaintextPassword=(?i)ConvertTo-SecureString\s*(?:-[S]tring)?\s*["']*(?P<secret>[^"')(?:]*)["']*
```

Comment:
>ConvertTo [](http://example.com/codeflow?start=36&length=9)

Do you have a test case you can add for this? Just add a pattern to the checked in data file for this rule.

Reply:
Here is the test file:

[https://github.com/microsoft/sarif-pattern-matcher/blob/main/Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_028.PlaintextPassword.ps1](https://github.com/microsoft/sarif-pattern-matcher/blob/main/Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_028.PlaintextPassword.ps1)

Reply:
@michaelcfanning Added the test from CodeAsData.